packages/components/src/components/common/toggleGroup/toggleGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group';
+
+import { cn } from '@/utils/class-name';
+import { toggleGroup } from './toggleGroup.css';
+
+type ToggleGroupProps = React.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Root> & {
+  gap?: 1 | 2 | 3 | 4 | undefined;
+};
+
+const ToggleGroup = React.forwardRef<React.ElementRef<typeof ToggleGroupPrimitive.Root>, ToggleGroupProps>(
+  ({ disabled, className, gap, ...props }, ref) => {
+    return (
+      <ToggleGroupPrimitive.Root
+        className={cn(toggleGroup({ gap }), className, 'ui-toggle-group')}
+        disabled={disabled}
+        {...props}
+        ref={ref}
+      />
+    );
+  }
+);
+ToggleGroup.displayName = ToggleGroupPrimitive.Root.displayName;
```

Comment:
as you no longer do something with the disabled prop, you can just remove it. Then it will be part of the ...props.
You also no longer do something else than return jsx so you can use the shorthand syntax:

```tsx
const ToggleGroup = React.forwardRef<React.ElementRef<typeof ToggleGroupPrimitive.Root>, ToggleGroupProps>(
  ({ className, gap, ...props }, ref) => (
    <ToggleGroupPrimitive.Root
      className={cn(toggleGroup({ gap }), className, 'ui-toggle-group')}
      {...props}
      ref={ref}
    />
  )
);
ToggleGroup.displayName = ToggleGroupPrimitive.Root.displayName;
```